src/glados/utils/automated_install.py

```
@@ -0,0 +1,129 @@
+    try:
+        subprocess.run("curl -fsSL https://ollama.com/install.sh | sh", shell=True, check=True)
+        print("Ollama installed successfully on Linux")
+    except subprocess.CalledProcessError:
+        raise RuntimeError("Failed to install Ollama on Linux")
+
+# This has not been tested yet !
+def install_ollama_macos():
+    """Install Ollama on macOS using Homebrew or direct download."""
+    try:
+        # Check if Homebrew is installed
+        if subprocess.run(["which", "brew"], capture_output=True, text=True).returncode == 0:
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)